azure-pipelines.yml

```
@@ -193,6 +198,89 @@ stages:
           condition: always()
 
 - ${{ if eq(variables._RunAsPublic, True) }}:
+
+  - stage: Test_CLI_Package_In_Helix_Android
+    displayName: Run CLI End-to-End Helix scenarios - Android
```

The other stages (and the directories) are all called "Integration tests". Maybe E2E is more fitting but I would just stick with one

```suggestion
    displayName: Run CLI Integration Helix scenarios - Android
```